components/ProfilePage/SocialMediaIcons.tsx

```diff
@@ -49,6 +53,22 @@ export const SocialMediaIcons = ({
             </External>
           </Col>
         )}
+
+        {blueSky && (
+          <Col>
+            <External plain href={`https://blulesky.org/${blueSky}`}>
```

Comment:
nit: I don't think this is the right BlueSky link - BlueSky profile links look like `https://bsky.app/profile/${name}`